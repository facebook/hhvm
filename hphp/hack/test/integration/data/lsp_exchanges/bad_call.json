[
{
  "jsonrpc": "2.0",
  "id": 1,
  "method": "textDocument/didOpen",
  "params": {
     "textDocument": {
       "uri":"${php_file_uri}",
       "languageId": "hack",
       "version":1,
       "text":"${php_file}"
     }
   }
},
{
  "jsonrpc": "2.0",
  "method": "$test/waitForHhServerReady",
  "params": {}
},
{
  "jsonrpc": "2.0",
  "id": 99,
  "method": "shutdown",
  "params": {}
}
]
