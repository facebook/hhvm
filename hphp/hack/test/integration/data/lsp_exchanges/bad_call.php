<?hh  //strict

function a_bad_call(): int {
  return b_bad_call();
}

function b_bad_call(): int {
  return 42;
}
