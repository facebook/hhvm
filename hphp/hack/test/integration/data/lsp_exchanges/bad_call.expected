[
  {
    "jsonrpc": "2.0",
    "id": 1,
    "error": {
      "code": -32002,
      "message": "Server not yet initialized",
      "data": {
        "stack": "Raised at file \"/data/users/fredemmott/fbsource/fbcode/hphp/hack/src/client/clientLsp.ml\", line 2119, characters 4-67\nCalled from file \"/data/users/fredemmott/fbsource/fbcode/hphp/hack/src/client/clientLsp.ml\", line 2378, characters 6-65\n"
      }
    }
  },
  {
    "jsonrpc": "2.0",
    "id": 99,
    "result": null
  }
]
